recipes/cytosim/meta.yaml

```diff
@@ -41,7 +41,8 @@ test:
 about:
   home: http://cytosim.org/
   license: GPL-3.0-or-later
-  license_family: GPL3
+  license_family: GPL3i
```

Comment:
```suggestion
  license_family: GPL3
```

Reply:
Can you please show me where `GPL3i` is defined? As far as I know there is only `GPL3` family.

Reply:
I resolved it because it is fixed, it is back to GPL3.

Reply:
The browser showed it as outdated now. OK.